lib/cancan/model_adapters/ar_outer_joins/ar_outer_joins.rb

```diff
@@ -0,0 +1,15 @@
+require "cancan/model_adapters/ar_outer_joins/join_builder"
+require "cancan/model_adapters/ar_outer_joins/join"
+
+module ArOuterJoins
+  def outer_joins(*joins)
```

Comment:
In Rails 5 we got `.left_joins`. The duplicate table in this regard was fixed in https://github.com/rails/rails/pull/30410.

Maybe rename this to `left_joins` and only use if it Rails version is below Rails 5.2?